arcgis-ios-sdk-samples/Augmented reality/Explore scenes in flyover AR/README.md

```diff
@@ -0,0 +1,45 @@
+# Explore scenes in flyover AR
+
+Use augmented reality (AR) to quickly explore a scene more naturally than you could with a touch or mouse interface.
+
+![Scene shown in an app](image1.png)
+
+## Use case
+
+You can use AR to drop into an area and visualize information, like a proposed development or a historical model of a city. You could use flyover AR to explore a city by walking through it virtually.
+
+## How to use the sample
+
+When you open the sample, you'll be viewing the scene from above. You can walk around, using your device as a window into the scene. Try moving vertically to get closer to the ground. Touch gestures which 
```

Comment:
It looks like the end of the paragraph was cut off.

Reply:
@nCastle1 Can you provide the rest of the text?  It is cut off in the README in your fork of the samples as well.

Reply:
Should just be:

> When you open the sample, you'll be viewing the scene from above. You can walk around, using your device as a window into the scene. Try moving vertically to get closer to the ground.

The statement about touch gestures was meant to be removed